src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/operators.ts

```diff
@@ -172,7 +205,10 @@ export async function getSuggestionsToRightOfOperatorExpression({
         suggestions.push(listCompleteItem);
       } else {
         const finalType = leftArgType || 'any';
-        const supportedTypes = getSupportedTypesForBinaryOperators(fnDef, finalType as string);
+        const supportedTypes =
+          operator.subtype === 'binary-expression'
```

Comment:
Now that we have redirected **is null** and **is not null** to the right ` branch condition` , if it passes through here, the tests crash without this check